hudi-kafka-connect/src/main/java/org/apache/hudi/connect/utils/KafkaConnectUtils.java

```diff
@@ -89,6 +140,23 @@ public static int getLatestNumPartitions(String bootstrapServers, String topicNa
    */
   public static Configuration getDefaultHadoopConf(KafkaConnectConfigs connectConfigs) {
     Configuration hadoopConf = new Configuration();
+
+    // add hadoop config files
+    if (!StringUtils.isNullOrEmpty(connectConfigs.getHadoopConfDir())
```

Comment:
Since, `DEFAULT_HADOOP_CONF_FILES` is being populated in the static block above, why not check `DEFAULT_HADOOP_CONF_FILES.isEmpty()`?

Reply:
@codope 
The default Hadoop configuration can solve the problem of a single environment, but we may also need to consider the need to manually configure `hadoop.conf.dir ` or ` hadoop.home` if different tasks need to write to different HDFS. 
So that I also separately added the parameters of hadoop environment config in `KafkaConnectConfigs`.